src/main/resources/interface-specification.yml

```diff
@@ -419,6 +419,56 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/events':
+    get:
+      tags:
+        - gateway
+      summary: Get list of events
+      description: Retrieves the list of events sent to the caller
+      operationId: getEventsFromId
+      security:
+        - bearerAuth: [ ]
+      parameters:
+        - name: lastEventId
+          in: query
+          description: returns organization events starting from this id
+          required: true
+          schema:
+            type: string
+        - name: limit
+          in: query
+          description: the number of events returned by this response
+          schema:
+            type: integer
+            format: int32
+            minimum: 1
+            maximum: 500
+            default: 100
+      responses:
+        '200':
+          description: Messages
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Messages'
+        '400':
+          description: Bad request
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '401':
+          description: Unauthorized
+          content:
+            application/problem+json:
+              schema:
+                $ref: '#/components/schemas/Problem'
+        '404':
```

Comment:
is this required?
I would expect an empty response for out of range `eventId`, or if not events exist for the given organization

Reply:
right.

Reply:
Here you get 404 only if the organization defined in the token does not exist.